packages/base/src/dialogs/symbology/colorRampUtils.ts

```diff
@@ -0,0 +1,136 @@
+import colormap from 'colormap';
+import colorScale from 'colormap/colorScale.js';
+import { useEffect } from 'react';
+
+import rawCmocean from '@/src/dialogs/symbology/components/color_ramp/cmocean.json';
+
+export interface IColorMap {
+  name: ColorRampName;
+  colors: string[];
+}
+
+const { __license__: _, ...cmocean } = rawCmocean as any;
```

Comment:
```suggestion
const { __license__: _, ...cmocean } = rawCmocean;
```

TypeScript is inferring the type no problem!